TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -64,42 +64,26 @@ class AllStopsTableViewController: UIViewController {
     }
 
     private func createSectionIndexesForBusStop() {
-        var currentChar: Character?
         var sectionIndexDictionary: [String: [Place]] = [:]
-        var currBusStopArray: [Place] = []
+        var numberBusStops: [Place] = []
 
-        currentChar = allStops.first?.name.capitalized.first
-
-        var numberBusStops: [Place] = {
-            if let firstStop = allStops.first {
-                return [firstStop]
-            }
-            return []
-        }()
-
-        if currentChar != nil {
-            for busStop in allStops {
-                if let firstChar = busStop.name.capitalized.first {
-                    if currentChar != firstChar {
-                        if !CharacterSet.decimalDigits.contains(currentChar!.unicodeScalars.first!) {
-                            sectionIndexDictionary["\(currentChar!)"] = currBusStopArray
-                            currBusStopArray = []
-                        }
-                        currentChar = firstChar
-                        currBusStopArray.append(busStop)
+        if !allStops.isEmpty {
+            allStops.forEach { busStop in
```

Comment:
Isn't the `if !allStops.isEmpty {` check meaningless here, since if `allStops` is empty, the `forEach` loop won't do anything?

Reply:
You right. Fixed!